internal/services/datafactory/data_factory_trigger_schedule_resource.go

```diff
@@ -53,6 +53,86 @@ func resourceDataFactoryTriggerSchedule() *pluginsdk.Resource {
 				ValidateFunc: validate.DataFactoryName(),
 			},
 
+			"description": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+
+			"schedule": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MinItems: 1,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"days_of_month": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type: pluginsdk.TypeInt,
+								ValidateFunc: validation.Any(
+									validation.IntBetween(1, 31),
+									validation.IntBetween(-31, -1),
+								),
+							},
+						},
+
+						"days_of_week": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							MaxItems: 7,
+							Elem: &pluginsdk.Schema{
+								Type:         pluginsdk.TypeString,
+								ValidateFunc: validation.IsDayOfTheWeek(false),
+							},
+						},
+
+						"hours": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type:         pluginsdk.TypeInt,
+								ValidateFunc: validation.IntBetween(0, 24),
+							},
+						},
+
+						"minutes": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type:         pluginsdk.TypeInt,
+								ValidateFunc: validation.IntBetween(0, 60),
+							},
+						},
+
+						"monthly_occurrence": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							MinItems: 1,
+							Elem: &pluginsdk.Resource{
+								Schema: map[string]*pluginsdk.Schema{
+									"day": {
+										Type:         pluginsdk.TypeString,
+										Required:     true,
+										ValidateFunc: validation.IsDayOfTheWeek(false),
+									},
+
+									"occurrence": {
```

Comment:
and occurrence is not clear, how about
```suggestion
									"week": {
```